iosMath/render/internal/MTTypesetter.m

```diff
@@ -628,6 +628,7 @@ - (void) createDisplayAtoms:(NSArray*) preprocessed
                     MTMathListDisplay* degree = [MTTypesetter createLineForMathList:rad.degree font:_font style:kMTLineStyleScriptScript];
```

Comment:
This comment is for line 524, GitHub won't let me add it there.

Do not fuse atoms if they have different colors. That will simplify the rendering below as each displayList can only have one color.